src/javascript/app_2/Stores/base_store.js

```diff
@@ -63,6 +65,21 @@ export default class BaseStore {
             writable  : true,
         });
 
+        const has_local_or_session_storage = local_storage_properties && local_storage_properties.length
+            || session_storage_properties && session_storage_properties.length;
+
+        if (has_local_or_session_storage) {
+            if (!store_name) {
+                throw new Error('store_name is required for local/session storage');
+            } else {
```

Comment:
@OskarAhl it doesn't need `else` because after throwing an error the code will no execute anymore.